_haikus/2022_First-Open-Source-Project.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: haiku
+title: Deep Sea King
+author: DHCJS
+---
+
+An old silent pond <br>
```

Comment:
This is my original Haiku

Reply:
The haiku you submitted is a famous one by the poet Basho

Reply:
Correct! I just google it